writers/postgres/messages.go

```diff
@@ -61,18 +81,18 @@ func (pr postgresRepo) Save(messages ...senml.Message) (err error) {
 		return
 	}()
 
-	for _, msg := range messages {
-		dbth, err := toDBMessage(msg)
+	for _, msg := range msgs {
+		id, err := uuid.NewV4()
```

Comment:
I was thinking we switched to ulid everyware ?

Reply:
Good catch. We did not, but we plan to do that. Since we need to do migrations to support ulid and maintain backward compatibility, I would prefer to do it in a separate pull request.